src/utils/model.js

```diff
@@ -1,157 +1,91 @@
-import Promise from 'bluebird';
 import * as moment from 'moment';
 import { useState, useEffect } from 'react';
-import { Projections } from './../models';
-import { STATES } from 'enums';
+import { Projections } from './../models/Projections';
+import { STATES, INTERVENTIONS } from 'enums';
 import DataUrlJson from 'assets/data/data_url';
 
 const DATA_URL = DataUrlJson.data_url.replace(/\/$/, '');
 
 async function fetchAll(urls) {
-  try {
-    const data = await Promise.all(
-      urls.map(url =>
-        fetch(url)
-          .then(response => {
-            // 404 files currently return an html page instead of 404,
-            // so need to handle this way...
-            return response.text();
-          })
-          .then(responseText => {
-            try {
-              const jsonResponse = JSON.parse(responseText);
-
-              return jsonResponse;
-            } catch (err) {
-              throw err;
-            }
-          }),
-      ),
-    );
-
-    return data;
-  } catch (error) {
-    throw error;
-  }
+  return Promise.all(
+    urls.map(async url => {
+      try {
+        const response = await fetch(url);
+        const textResponse = await response.text();
+        return JSON.parse(textResponse);
+      } catch {
+        return null;
```

Comment:
😬 

Why do we swallow errors?  At minimum, can we `console.error()` the error?

Reply:
Purposeful though open to being convinced otherwise :). The idea here is that a missing projection file is an expected way of indicating that that projection is invalid / missing at the moment.  Eg. if a given set of projections is missing inference, this will be null (and is handled correctly throughout).